draft-ietf-ppm-dap.md

```diff
@@ -1214,6 +1214,17 @@ initialization step for preparation. The Helper's response, along with each
 subsequent request and response, carries the remaining messages exchanged during
 preparation.
 
+Leader to Helper HTTP communication can be performed asynchronously or
+synchronously. In the asynchronous case, the Leader sends aggregation job HTTP
+requests to the Helper that can be handled quickly. In the synchronous case, the
+Helper processes the aggregation job while holding the request open, reducing
+the number of HTTP requests that are sent. This choice allows a Helper
```

Comment:
I think we can define "synchronous" and "asynchronous" more clearly.
```suggestion
Normally the Helper process each step synchronously, meaning it computes each step of the computation before producing a response to the Leaders HTTP request. The Helper also has the option to process each step asynchronously. This means the Helper responds to the request immediately, without producing the next step of the computation. To continue, the Leader polls the Helper until it responds with the next steep. This choice allows a Helper
```

Reply:
SGTM. I wordsmithed a bit and moved it down in this section, so that we're not introducing concepts that haven't been formally introduced yet (i.e. `steps`).